src/app/containers/ImageWithPlaceholder/index.jsx

```diff
@@ -65,6 +65,7 @@ const ImageWithPlaceholder = ({
     <ImageComponent onLoad={() => setIsLoaded(true)} {...imageProps} />
   );
   const shouldPreload = !isAmp && preload;
+  const isHero = isAmp && preload;
```

Comment:
Canonical images can be hero images too so the naming of this var is not quite correct and might cause confusion when reading the logic without looking at the definition. I'd maybe just do `{...(isAmp && preload && { 'data-hero': true })};` or rename the var to something like `isAmpHeroImage`